[BUG] Developmental build https://liquidbounce.net/download/10821 Incompatible with Lunar client

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

v0.35.3

### Operating System

Windows 11

### Minecraft Version

1.21.11

### Describe the bug

I was trying to load an experimental build of Liquid Bounce as I heard they had made a Big jump vs. the Stable Builds. I decided to load it in lunar client as I saw in the Readme.txt It can be done. 

Main Issue:
When launching in Lunar client into minecraft version 1.21.11
LiquidBounce, Works and the game loads, but instantly after TRYING to interact with the game, it crashes as it tries to pull a cosmetic but Lunar denies the request






### Steps to reproduce

- Use Lunar client latest available version
- Go into versions And use minecraft 1.21.11 with the fabric loader 0.18.2
- Put Latest Developmental Build of Liquid Bounce v0.35.3 Nextgen
- Run

### Client Log

```shell
Reported exception thrown!
14:04:51
net.minecraft.class_148: Rendering entity in world
14:04:51
	at Genesis/net.minecraft.class_898.method_72976(SourceFile:180) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_898.bridge$renderEntityWithPosYaw(SourceFile:749) ~[?:?]
14:04:51
	at Genesis/com.moonsworth.lunar.client.HCCOHCIIRIICCRCICOHOHCHIHOIROR.RCCIICRIHOORHOCHORCHCOHRICORCI.OCHHOCIHIIIORHCHIICHCOHHRIHHOI.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.RCCIICRIHOORHOCHORCHCOHRICORCI(Unknown Source) ~[CRIRIOCOIRHCRCHIICIHHCRRROOHHO.class:?]
14:04:51
	at Genesis/com.moonsworth.lunar.client.HCCOHCIIRIICCRCICOHOHCHIHOIROR.RCCIICRIHOORHOCHORCHCOHRICORCI.OCHHOCIHIIIORHCHIICHCOHHRIHHOI.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.HIIOOCCROOCCHHIORIIOOICRIORCHC(Unknown Source) ~[CRIRIOCOIRHCRCHIICIHHCRRROOHHO.class:?]
14:04:51
	at Genesis/com.moonsworth.lunar.client.HCCOHCIIRIICCRCICOHOHCHIHOIROR.RCCIICRIHOORHOCHORCHCOHRICORCI.OCHHOCIHIIIORHCHIICHCOHHRIHHOI.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.HIIOOCCROOCCHHIORIIOOICRIORCHC(Unknown Source) ~[CRIRIOCOIRHCRCHIICIHHCRRROOHHO.class:?]
14:04:51
	at Genesis/com.moonsworth.lunar.client.HCCOHCIIRIICCRCICOHOHCHIHOIROR.RCCIICRIHOORHOCHORCHCOHRICORCI.OCHHOCIHIIIORHCHIICHCOHHRIHHOI.CCOCOIHOCIOOHHOIOHRRIHHROOCICR.HIIOOCCROOCCHHIORIIOOICRIORCHC(Unknown Source) ~[CCOCOIHOCIOOHHOIOHRRIHHROOCICR.class:?]
14:04:51
	at Genesis/com.moonsworth.lunar.client.HCCOHCIIRIICCRCICOHOHCHIHOIROR.RCCIICRIHOORHOCHORCHCOHRICORCI.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.COHRHOORIORCOOOCHCHIICIHOCHHHC(Unknown Source) ~[CRIRIOCOIRHCRCHIICIHHCRRROOHHO.class:?]
14:04:51
	at Genesis/com.moonsworth.lunar.client.HCCOHCIIRIICCRCICOHOHCHIHOIROR.RCCIICRIHOORHOCHORCHCOHRICORCI.HIIOOCCROOCCHHIORIIOOICRIORCHC.RIOCIRCHHIOORCCOIICRIOCCRIRHCC(Unknown Source) ~[HIIOOCCROOCCHHIORIIOOICRIORCHC.class:?]
14:04:51
	at Genesis/com.moonsworth.lunar.client.HCCOHCIIRIICCRCICOHOHCHIHOIROR.RCCIICRIHOORHOCHORCHCOHRICORCI.HIIOOCCROOCCHHIORIIOOICRIORCHC.HIIOOCCROOCCHHIORIIOOICRIORCHC(Unknown Source) ~[HIIOOCCROOCCHHIORIIOOICRIORCHC.class:?]
14:04:51
	at Genesis/com.moonsworth.lunar.CCOHHRRRIHRCHIRIOCCCHOIOCOIHCR.RIIHIRCHRRHCIIOCIIIIIOICOROHIO.CCOHHRRRIHRCHIRIOCCCHOIOCOIHCR.RCCIICRIHOORHOCHORCHCOHRICORCI.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.HIIOOCCROOCCHHIORIIOOICRIORCHC(Unknown Source) ~[CRIRIOCOIRHCRCHIICIHHCRRROOHHO.class:?]
14:04:51
	at Genesis/com.moonsworth.lunar.CCOHHRRRIHRCHIRIOCCCHOIOCOIHCR.RIIHIRCHRRHCIIOCIIIIIOICOROHIO.CCOHHRRRIHRCHIRIOCCCHOIOCOIHCR.RCCIICRIHOORHOCHORCHCOHRICORCI.CRIRIOCOIRHCRCHIICIHHCRRROOHHO.HIIOOCCROOCCHHIORIIOOICRIORCHC(Unknown Source) ~[CRIRIOCOIRHCRCHIICIHHCRRROOHHO.class:?]
14:04:51
	at Genesis/net.minecraft.class_11228.md056a41$lambda$lunar$afterRender$1$1(SourceFile:849) ~[?:?]
14:04:51
	at Genesis/com.moonsworth.lunar.CCOHHRRRIHRCHIRIOCCCHOIOCOIHCR.RIIHIRCHRRHCIIOCIIIIIOICOROHIO.OCHIOOORICCCROHOIRHCCRHCCIRICH.IOCCIHRRRORCOROROIROORHCROIIIH.RHHORCHCIOOHROCRIRRIIIIOCHRROO(Unknown Source) ~[IOCCIHRRRORCOROROIROORHCROIIIH.class:?]
14:04:51
	at Genesis/net.minecraft.class_11228.handler$zzj000$lunar$afterRender(SourceFile:842) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_11228.method_70890(SourceFile:184) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_757.method_3192(SourceFile:585) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_310.method_1523(SourceFile:1393) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_310.method_1514(SourceFile:966) [?:?]
14:04:51
	at Genesis/net.minecraft.client.main.Main.main(SourceFile:250) [Main.class:?]
14:04:51
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
14:04:51
	at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
14:04:51
	at com.moonsworth.lunar.genesis.ClientGameBootstrap.apply(Unknown Source) [genesis-0.1.0-SNAPSHOT-all.jar:?]
14:04:51
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
14:04:51
	at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
14:04:51
	at com.moonsworth.lunar.genesis.RCCIICRIHOORHOCHORCHCOHRICORCI.RCCIICRIHOORHOCHORCHCOHRICORCI(Unknown Source) [genesis-0.1.0-SNAPSHOT-all.jar:?]
14:04:51
	at com.moonsworth.lunar.genesis.Genesis.run(Unknown Source) [genesis-0.1.0-SNAPSHOT-all.jar:?]
14:04:51
	at com.moonsworth.lunar.genesis.Genesis.main(Unknown Source) [genesis-0.1.0-SNAPSHOT-all.jar:?]
14:04:51
Caused by: java.lang.NullPointerException
14:04:51
	at Genesis/net.ccbluex.liquidbounce.features.cosmetic.CosmeticService.fetchCosmetic(CosmeticService.kt:209) ~[CosmeticService.class:?]
14:04:51
	at Genesis/net.ccbluex.liquidbounce.features.cosmetic.CosmeticService.fetchCosmetic$default(CosmeticService.kt:96) ~[CosmeticService.class:?]
14:04:51
	at Genesis/net.ccbluex.liquidbounce.features.cosmetic.CosmeticService.getCosmetic(CosmeticService.kt:148) ~[CosmeticService.class:?]
14:04:51
	at Genesis/net.ccbluex.liquidbounce.features.cosmetic.CosmeticService.hasCosmetic(CosmeticService.kt:166) ~[CosmeticService.class:?]
14:04:51
	at Genesis/net.minecraft.class_978.modifyExpressionValue$dho000$liquidbounce$onRender(SourceFile:537) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_978.method_4181(SourceFile:25) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_978.method_4199(SourceFile:15) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_922.method_4054(SourceFile:113) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_922.method_3936(SourceFile:39) ~[?:?]
14:04:51
	at Genesis/net.minecraft.class_898.method_72976(SourceFile:154) ~[?:?]
14:04:51
	... 26 more
```

### Screenshots

_No response_

--- a/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/render/entity/feature/MixinDeadmau5EarsLayer.java
+++ b/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/render/entity/feature/MixinDeadmau5EarsLayer.java
@@ -34,7 +34,10 @@ public class MixinDeadmau5EarsLayer {
 
     @ModifyExpressionValue(method = "submit(Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/SubmitNodeCollector;ILnet/minecraft/client/renderer/entity/state/AvatarRenderState;FF)V", at = @At(value = "FIELD", target = "Lnet/minecraft/client/renderer/entity/state/AvatarRenderState;showExtraEars:Z", remap = false))
     private boolean onRender(boolean original, @Local(argsOnly = true) AvatarRenderState playerEntityRenderState) {
-        return original || CosmeticService.INSTANCE.hasCosmetic(((EntityRenderStateAddition) playerEntityRenderState).liquid_bounce$getEntity().getUUID(), CosmeticCategory.DEADMAU5_EARS);
+        if (original) return true;
+
+        var entity = ((EntityRenderStateAddition) playerEntityRenderState).liquid_bounce$getEntity();
+        return entity != null && CosmeticService.INSTANCE.hasCosmetic(entity.getUUID(), CosmeticCategory.DEADMAU5_EARS);
     }
 
 }

